packages/paste-core/components/form-pill-group/stories/index.stories.tsx

```diff
@@ -18,10 +18,46 @@ export const Basic: React.FC = () => {
         <FormPill {...pillState} selected data-testid="form-pill-selected">
           Baseball
         </FormPill>
-        <FormPill {...pillState}>Football</FormPill>
+        <FormPill {...pillState}>
+          <Avatar size="sizeIcon10" name="avatar example" src="./avatars/avatar4.png" />
+          Football
+        </FormPill>
         <FormPill {...pillState} selected>
+          <Avatar size="sizeIcon10" name="avatar example" src="./avatars/avatar4.png" />
           Soccer
         </FormPill>
+        <FormPill {...pillState} variant="error">
+          Pizza
+        </FormPill>
+      </FormPillGroup>
+    </form>
+  );
+};
+
+export const Disabled: React.FC = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup {...pillState} data-testid="form-pill-group" aria-label="Your favorite sports:">
+        <FormPill data-testid="form-pill" {...pillState} disabled>
+          <CalendarIcon decorative size="sizeIcon10" />
```

Comment:
TODO: I need to rethink these story names and which ones display errors, disabled, avatars, etc. to show all the new permutations cleanly.